src/test/rpc/JSONRPC_test.cpp

```diff
@@ -79,11 +78,11 @@ R"({
         "TransactionType": "Payment"
     }
 })",
-{
+{{
```

Comment:
I know this has been fixed in newer clangs, but older clangs used to issue a warning about initializing `std::array` with just a single set of braces. I vote to remove the double braces, as you've done here, but wanted to flag the issue in case you're worried about warnings from older compilers. This is more of a concern if we change our CMake to error on warnings by default.

Reply:
I switched to double braces because the macOS version of clang warns without that, and I have my build set to treat warnings as errors.  Also I think most, if not all, of the CI builds treat warnings as errors.  So I think the double brace initialization of `std::array` will be necessary for some time to come.

Reply:
I looked too quickly and misread the change. I thought this went from double to single braces, not single to double. We can easily fix any CI build failures, as we can install whatever clang version we want. If Mac's clang isn't new enough to fix the warning that's a bigger issue and we should compile warning free there too. However, I'd like to switch to single braces for new code as soon as Mac's clang is new enough to fix the warning.